analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -283,6 +309,8 @@ static void MultiplePop()
         class MyClass
         {
             public string MyProperty { get; set; }
+            public string MyField;
```

Comment:
`const` is also a field. Add these const fields:
* An value type `int`
* A string `"Some text"`
* An object "null" (only possible value)

You need to learn constraints before you can test like e.g. so:
```cs
if (oConst == null)
{
    oConst.ToString() // Should raise
}
// or
_ = sConst.Length; // learns sConst is not null
//